shell/src/main/java/org/apache/accumulo/shell/commands/GrepCommand.java

```diff
@@ -63,6 +66,12 @@ public int execute(final String fullCommand, final CommandLine cl, final Shell s
       if (cl.hasOption(negateOpt.getOpt())) {
         negate = true;
       }
+
```

Comment:
The super class `ScanCommand` has a protected method called `addScanIterators()`.  I am not completely sure, but I think you may be able to call it to setup the profile iters.  If so, very little changes would be needed for this class.

Reply:
Thanks for the suggestion.  I had to make a small change to addScanIterators signature to accept ScannerBase instead of Scanner, but it removed most of my code update, so I hope you are ok with the change.

Reply:
Also, I re-ran the test I documented in the ticket and I got the same results with this new update.